src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_js.html.twig

```diff
@@ -1,12 +1,12 @@
 <div id="sfwdt{{ token }}" class="sf-toolbar sf-display-none" role="region" aria-label="Symfony Web Debug Toolbar">
-    {% include('@WebProfiler/Profiler/toolbar.html.twig') with {
+    {{ include('@WebProfiler/Profiler/toolbar.html.twig') with {
```

Comment:
I think this one cannot work, the with should be passed as an argument to the include function instead.

Reply:
Thanks, fixed.